### DISCUSSION TITLE
What is the latest version

### DISCUSSION THREAD

**Discussion by maidamai0** [microsoft/WindowsAppSDK#1946] [Q&A] [answered]
The **readme.md** implies the latest version is `1.0`:
> 1.0 is available today and includes unpackaged support with WinUI 3, single-project templates, and more!

But the latest release version is [0.86](https://github.com/microsoft/WindowsAppSDK/releases/tag/v0.8.6.0):
> 0.8.6 is now LIVE!

It is very confusing I must say.

**Comment by ghost1372** [comment]
https://github.com/microsoft/WindowsAppSDK/discussions/1931#discussioncomment-1872310

**Comment by andrewleader** [comment, marked as answer]
`1.0.0` is the latest stable version. We released a servicing update to the `0.8.x` version chain after releasing 1.0, and GitHub Releases sort by newest rather than version number, so `0.8.6` shows up as "latest", even though it's older in functionality. **1.0 is what you should be using** unless you were previously using 0.8 and aren't yet able to invest the time to update to 1.0.
